scripts/rebuild-testbench.sh

```diff
@@ -5,16 +5,27 @@
 # stop on most errors
 set -e
 
-SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
-SOF_REPO=$(dirname "$SCRIPT_DIR")
-TESTBENCH_DIR="$SOF_REPO"/tools/testbench
+# check if we are in the sof FW directory
+if [ -f "Kconfig.sof" ]; then
+    # now got to parent sof workspace directory
+    cd ..
```

Comment:
Isn't this is a bit surprising to change the working directory when running the script? I don't see CWD restored at end. Wouldn't a less surprising method be to error if not run in correct directory?

PS and s/got to/go to/

Reply:
Fixed the go to.
Expectation is that the script will be run from FW directory or the parent SOF workspace only. The directory change done by the script (in its own shell) will not change the users directory in their shell.